### ISSUE TITLE
Separation of Documents

### ISSUE THREAD

**Issue by SteVwonder** [ExaWorks/job-api-spec#70]
We've mentioned a few times in various threads that eventually we could end up with a few different documents.  I wanted to open an issue to track A) which different documents we'd ultimately like to end up with and B) how and when we plan to get to that number/breakdown.

In the various discussions, I've seen a few separate documents proposed:

1. The Standard (language-agnostic?)
2. Language Specification APIs (e.g., python, scala, java, C)
3. Resource and Job Serialization Formats (currently an appendix to the Standard)
4. Examples (currently an appendix to the Standard)

**Comment by andre-merzky** [comment]
Ack on (1), (2) and (3).  (1) and (2) depend a bit on #71.  Keeping (3) separate simplifies versioning of the job spec, which is nice.

I am not sure if (4) should be a separate document.  What would it contain beyond the examples we already do or likely will include in (1) to (3)?

**Comment by SteVwonder** [comment]
> What would it contain beyond the examples we already do or likely will include in (1) to (3)?

Agreed that a separate examples doc is probably not going to be necessary due to 1 and 2.
